bitacross-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -155,7 +156,8 @@ impl TrustedCallSigning<TrustedCallSigned> for TrustedCall {
 		payload.append(&mut mrenclave.encode());
 		payload.append(&mut shard.encode());
 
-		TrustedCallSigned { call: self.clone(), nonce, signature: pair.sign(payload.as_slice()) }
+		// use blake2_256 hash to shorten the payload - see `verify_signature` below
+		TrustedCallSigned { call: self.clone(), nonce, signature: pair.sign(&blake2_256(&payload)) }
```

Comment:
The size of the signature remains same, I think we are replacing the computation of signing a large message with the computation of producing the hash for a large message. I do prefer hashing a message before signing it for consistency.

Reply:
Please ignore this, It was rhetorical